index.html

```diff
@@ -0,0 +1,183 @@
+          <li>Programe</li>
+          <li>Join</li>
+          <li>Sponser</li>
+          <li>News</li>
+          <li><a class="highlight" href="#" id="top-icon">Microsoft</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <!-- hero section -->
+      <section class="hero">
+        <nav class="mobile">
+          <div class="burger">
+            <span></span>
+            <span></span>
+            <span></span>
+          </div>
+          <div class="mobile-menu">
+            <ul class="lin-parent">
+              <li><a href="./index.html">Home</a></li>
+              <li><a href="./pages/about.html">About</a></li>
+              <li>Programe</li>
+              <li>Join</li>
+              <li>Sponser</li>
+              <li>News</li>
+              <a class="highlight" href="#" id="active-link">Microsoft</a>
+            </ul>
+            <div class="close">
+              <img src="./images/home/XIcon.svg" alt="close" />
+            </div>
+          </div>
+        </nav>
+        <div class="inner">
+          <span>"Hello! Sharing world"</span>
+          <h1>
+            creative commons <br />
+            global summit <br />
+            2022
+          </h1>
+          <div class="box-desc">
+            <p>
+              A joy celebration beliving in the value of openness and sharing,
+              creating <br />
+              a postive change with people over 80 countries is taking place in
+              October, in korea
+            </p>
+          </div>
+          <div class="info">
+            <p class="history">2022.10.15(THU) ~ 16(FRI)</p>
+            <p class="copy-right">
+              @ National Museum of korea, Art Center Nabi and more
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <!-- main programe section -->
+      <section class="main-programe">
+        <h3 class="sec-title">Main Programe</h3>
+        <article class="cards-group">
+          <div class="card">
+            <img src="./images/home/research.svg" alt="icon" />
+            <h4 class="card-title">Lecture</h4>
+            <p class="card-desc">
+              listen to the speakers from various countries about the message of
+              sharing and opening.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/design.svg" alt="icon" />
+            <h4 class="card-title">CC Exhibition</h4>
+            <p class="card-desc">
+              Appreciate various creations applying CC licence of
+              artists,organized from Art center Nabi.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/networking.svg" alt="network-icon" />
+            <h4 class="card-title">Forum</h4>
+            <p class="card-desc">
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please pay attention to your `Main Program` section elements because the distance between `texts` is not available and the `position` is different, please do it as shown in Behance's design 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/232789606-914bbd6f-bde2-4af5-8626-3da3a1923d63.png)

Reply:
@mukhammadrizooff error has fixed and align the text according to Behance design .